data/offsides/meta.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: offsides
+description: |-
+    OffSIDES is a database of individual drug side effect
+    signals mined from the FDA's Adverse Event Reporting System. The
+    innovation of OffSIDES is that a propensity score matching (PSM) model
+    is used to identify control drugs and produce better PRR estimates. In
+    OffSIDES we focus on drug safety signals that are not already
+    established by being listed on the structured product label - hence
+    they are off-label drug side effects.
+targets:
+    - id: PRR
+      description: proportional reporting ratio
+      type: continuous
+      names:
+          - proportional reporting ratio
+    - id: PRR_error
+      description: standard error of the PRR estimate
+      type: continuous
+      sample: false
+      names:
+          - standard error of the proportional reporting ratio error
+    - id: mean_reporting_frequency
+      description: mean reporting frequency for the drug
+      type: continuous
+      names:
+          - mean reporting frequency
+identifier:
+    - id: drug_concept_name
+      description: RxNorm name string for the drug
+      type: categorical
+    - id: condition_concept_name
+      description: MedDRA identifier for the side effect
+      type: categorical
```

Comment:
this will need to use prompt templates, I guess, because they are not independent.